### PR TITLE
Fix block suggester positioning

### PR DESCRIPTION
**Pull request by luisbettencourt**
## 🌟 What is the purpose of this PR?

The block and mention suggesters can overflow outside when opening at the bottom of the document. Also, scrolling after opening them causes them to float in the same place instead of scrolling with the document.

before

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1201095311341924/1202708815111578/f) _(internal)_

## 🔍 What does this change?

- Changed the way the suggesters are positioned in the document by using a mui Popper.

## ❓ How to test this?

Open mention ( @ ) and block ( / )  suggesters and scroll around the page.

## 📹 Demo

after

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202010891739489

### CODE REVIEW

File: packages/hash/frontend/src/blocks/page/PageBlock.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export const PageBlock: FunctionComponent<PageBlockProps> = ({
   return (
     <UserBlocksProvider value={blocks}>
       <BlockLoadedProvider routeHash={routeHash}>
-        <div id="root" ref={root} />
+        <div id="root" ref={root} style={{ position: "relative" }} />
```
Comment (teenoh) [inline packages/hash/frontend/src/blocks/page/PageBlock.tsx:86]: Nit: Let's use MuiBox here and add the style via the sx prop

File: packages/hash/frontend/src/blocks/page/createSuggester/Suggester.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const Suggester = <T,>({
   renderItem,
   error,
   sx = [],
-}: SuggesterProps<T>): ReactElement => {
```
Comment (teenoh) [inline packages/hash/frontend/src/blocks/page/createSuggester/Suggester.tsx:28]: Why was this removed?

**Review by teenoh — APPROVED** [review]
Thanks @luisbettencourt. Tested and this works as expected

File: packages/hash/frontend/src/blocks/page/PageBlock.tsx (hunk 1 above)
Comment (luisbettencourt) [reply packages/hash/frontend/src/blocks/page/PageBlock.tsx]: Done in 30521136de117ad64c8a37ddeca6f71bcab0d874

**Review by teenoh — APPROVED** [review]
